### PR TITLE
minor fix for document

### PR DESCRIPTION
**Pull request by LiYing2010**
### Change List
minor fix for documents: typo, code style

### CODE REVIEW

**Review by IgnatBeresnev — COMMENTED** [review]
Wow, thanks for fixing a bunch of small inconsistencies, I would've never noticed these!

File: docs/topics/dokka-plugins.md [hunk 1]
```diff
@@ -112,13 +112,13 @@ If you are using [JSON configuration](dokka-cli.md#run-with-json-configuration),
...
-    "./mathjax-plugin-%dokkaVersion%.jar"
-  ],
-  ...
+    ...
+    "pluginsClasspath": [
+        "./dokka-base-%dokkaVersion%.jar",
+        "...",
+        "./mathjax-plugin-%dokkaVersion%.jar"
+    ],
+    ...
```
[the remark this answers is not in the document]
Comment (LiYing2010) [reply docs/topics/dokka-plugins.md:121]: OK
I will revert it back to 2 spaces

File: docs/topics/dokka-plugins.md [hunk 2]
```diff
@@ -191,14 +191,14 @@ import org.jetbrains.dokka.gradle.DokkaTask
...
+        "customAssets": ["${file("assets/my-image.png")}"],
+        "customStyleSheets": ["${file("assets/my-styles.css")}"],
+        "footerMessage": "(c) 2022 MyOrg"
     }
     """
     pluginsMapConfiguration.set(
-            // fully qualified plugin name to json configuration
-            ["org.jetbrains.dokka.base.DokkaBase": dokkaBaseConfiguration]
+        // fully qualified plugin name to json configuration
+        ["org.jetbrains.dokka.base.DokkaBase": dokkaBaseConfiguration]
```
[the remark this answers is not in the document]
Comment (LiYing2010) [reply docs/topics/dokka-plugins.md:201]: no special reason
I just found that all code in groovy example are using 4 spaces ident,
and these 2 lines are the only exception which using 8 spaces ident


> that's how IntelliJ refactors groovy code by default on my machine

you are right
looks like IntelliJ recognize it as `Continuation ident`, and the default setting for it is 8

I will revert it back soond

File: docs/topics/dokka-plugins.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (LiYing2010) [reply docs/topics/dokka-plugins.md]: done
I reverted it

File: docs/topics/dokka-plugins.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LiYing2010) [reply docs/topics/dokka-plugins.md]: done
I reverted it

**Comment by LiYing2010** [comment]
@IgnatBeresnev 

I have reverted changes for idents,
but there are 2 commits in this PR now,
do you want me rebase and force-push this PR, so I can make the commit history more beautiful?

**Review by IgnatBeresnev — APPROVED** [review]

**Comment by IgnatBeresnev** [comment]
Once again, thank you! :+1: :+1: 

> I have reverted changes for idents,
> but there are 2 commits in this PR now,
> do you want me rebase and force-push this PR, so I can make the commit history more beautiful?

We use GitHub's "Squash and merge", so there will be a single merge commit anyway, no need to do anything yourself, but thanks for asking and willing to do it :)

**Comment by LiYing2010** [comment]
Thank you
I am glad I can help a little

**Pull request merged**